### DISCUSSION TITLE
My edges get intersected

### DISCUSSION THREAD

**Discussion by traineerookie** [xyflow/xyflow#3137] [Q&A]
Hi In my react flow project. I'm trying to use a node which is binary node where it has two source handles like yes or no where nodes will be added below the node like i could add a new node below the source handle edge of yes or on no. But the issue is if i add below a node below yes means my YES edge is basically in the left it goes to the right and the right edges comes to the right.

[imagereference](https://imgur.com/a/iYnfF9X)

for layout I'm using dagre the same sample code used in react flow documentation

**Comment by OsamaKhalilMohamed** [comment]
having the same issue, any solutions yet?
